src/components/layout/Navigation.tsx

```diff
@@ -36,15 +36,15 @@ export const Navigation: React.FC<Props> = ({ me, hideSearch = false }) => {
       <MobileNav user={me} />
       <div
         className={cn(
-          'max-md:hidden max-h-20 bg-grey-600 bg-opacity-85 backdrop-blur-[25px] w-full flex justify-center',
+          'max-md:hidden max-h-20 bg-grey-600 bg-opacity-85 backdrop-blur-[25px] flex w-full px-6 lg:px-8 justify-center',
```

Comment:
is this working as expected? I did some change around this yesterday and it was looking as expected

Reply:
it is working as expected on Discover screen which has no paddings in its main content container

Reply:
I think we should make our Navigation to have styling per Figma (it can be that some stylings are wrong in the app since the navbar has been redesigned a couple of times). And then adjust the `Body` according to the `Navigation`. Does it makes sense?

Reply:
Yes, I think this is part of that process since Mattan told me to remove it but is aware that it currently doesnt match Figma.

Reply:
So yeah I would suggest to make Navigation to match with Figma and then we can update all screens accordingly.